### PR TITLE
[MAINT] Add missing changelog entries

### PR DESCRIPTION
**Pull request by tsbinns**
Updates the `whats_new` page with those [changes since v0.7 release](https://github.com/mne-tools/mne-connectivity/compare/v0.7...main).

### CODE REVIEW

File: doc/whats_new.rst [hunk 1]
```diff
@@ -19,20 +19,37 @@ Here we list a changelog of MNE-connectivity.
 Version 0.8 (in dev)
 --------------------
 
+Functionality now requires ``Python >= 3.10``.
```
Comment (drammock) [inline doc/whats_new.rst:22]:
```suggestion
Minimum supported Python version is now 3.10.
```

File: doc/whats_new.rst [hunk 2]
```diff
@@ -19,20 +19,37 @@ Here we list a changelog of MNE-connectivity.
...
+
 Enhancements
 ~~~~~~~~~~~~
 
-
+- Add a new :class:`~mne_connectivity.decoding.CoherencyDecomposition` class for decomposing connectivity sources using multivariate coherency-based methods, by `Thomas Binns`_ (:pr:`193`).
+- Add new plotting methods :meth:`~mne_connectivity.decoding.CoherencyDecomposition.plot_filters` and :meth:`~mne_connectivity.decoding.CoherencyDecomposition.plot_patterns` to the :class:`~mne_connectivity.decoding.CoherencyDecomposition` class for visualising the decomposed connectivity sources, by `Thomas Binns`_ (:pr:`208`).
```
Comment (drammock) [inline doc/whats_new.rst:28]: It's fine as-is, but FYI you could hack the cross-refs to be a bit more contextually informative, like this:

```suggestion
- Add new plotting methods :meth:`CoherencyDecomposition.plot_filters() <mne_connectivity.decoding.CoherencyDecomposition.plot_filters>` and :meth:`CoherencyDecomposition.plot_patterns() <mne_connectivity.decoding.CoherencyDecomposition.plot_patterns>` for visualising the decomposed connectivity sources, by `Thomas Binns`_ (:pr:`208`).
```

File: doc/authors.inc [hunk 3]
```diff
@@ -16,3 +16,4 @@
 .. _Richard Köhler: https://github.com/richardkoehler
 .. _Mohammad Orabe: https://github.com/orabe
 .. _Mina Jamshidi: https://github.com/minajamshidi
+.. _Marijn van Vliet: https://github.com/wmvanvliet
```
Comment (drammock) [inline doc/authors.inc:19]: missing newline at EOF

```suggestion
.. _Marijn van Vliet: https://github.com/wmvanvliet

```

File: doc/whats_new.rst [hunk 4]
```diff
@@ -19,20 +19,37 @@ Here we list a changelog of MNE-connectivity.
...
+- Add support for computing multiple components of multivariate connectivity in the :func:`~mne_connectivity.spectral_connectivity_epochs` and :func:`~mne_connectivity.spectral_connectivity_time` functions and :class:`~mne_connectivity.decoding.CoherencyDecomposition` class, and add support for storing data with a components dimension in all :class:`~mne_connectivity.Connectivity` classes, by `Thomas Binns`_ and `Eric Larson`_ (:pr:`213`).
+- Add support for :class:`mne.time_frequency.EpochsSpectrum` objects to be passed as data to the :func:`~mne_connectivity.spectral_connectivity_epochs` function, by `Thomas Binns`_ and `Eric Larson`_ (:pr:`220`).
+- Update the cross-references for relevant function and classes and make data types more explicit throughout the documentation, by `Thomas Binns`_ (:pr:`214`).
+- Improve the documentation of the ``fmin`` and ``cwt_freqs`` parameters in the :func:`~mne_connectivity.spectral_connectivity_epochs` function, by `Richard Köhler`_ and `Daniel McCloy`_ (:pr:`242`).
```
Comment (drammock) [inline doc/whats_new.rst:32]: I'd probably put this under "Bug", generally if the docs are wrong I consider it a bugfix to correct them.

**Review by drammock — COMMENTED** [review]
thanks for correcting this!

**Comment by tsbinns** [comment]
Tnanks for the review @drammock! That should be everything updated.

**Review by drammock — APPROVED** [review]

**Comment by drammock** [comment]
thanks again @tsbinns!

**Pull request merged**